btree_tests/ycsb.cpp

```diff
@@ -346,14 +346,7 @@ void ycsb_load_run_randint(int index_type, int wl, int kt, int ap, int num_threa
                         // printf("insert key %lu\n", keys[i]);
                         concurrent_map.insert({keys[i], keys[i]});
                     } else if (ops[i] == OP_READ) {
-                        if( concurrent_map.value(keys[i]) != keys[i]) {
-#if LEAFDS
-                            // std::cout << "[BP-TREE] wrong key read: " << concurrent_map.value(keys[i]) << " expected:" << keys[i] << std::endl;
-#else
-                            // std::cout << "[BTREE] wrong key read: " << concurrent_map.value(keys[i]) << " expected:" << keys[i] << std::endl;
-#endif
-                            // exit(0);
-                        }
+                        concurrent_map.value(keys[i]);
```

Comment:
what is this doing? aren't you just performing a lookup and throwing away the result.
The optimizer can just throw this away since it is calling a const function